projects/common/src/color/color.service.ts

```diff
@@ -41,6 +41,13 @@ export class ColorService {
     return rgb(colorHex).brighter(basis).formatHex();
   }
 
+  public adjustOpacity(color: string, basis: number): string {
+    const rgbColor = rgb(color);
+    rgbColor.opacity = basis;
+
+    return rgbColor + '';
```

Comment:
instead of this, you can do `return rgb(rgbColor.r, rgbColor.g, rgbColor.b, opacity).toString`, I think.

little cleaner than the implicit string conversion at L48 and setting opacity.

Reply:
fixed